iotfunctions/metadata.py

```diff
@@ -2517,6 +2520,9 @@ def __init__(self, name, estimator, estimator_name, params, features, target, ev
             self.expiry_date = None
         self.viz = {}
 
+    def hasStdDev(self, pred_stddev):
```

Comment:
use python naming conversions. hasStdDev --> has_std_dev also better to use full name standard_deviation instead of std_dev.

Reply:
done